ShellUnitTest/gTestCmdFullWrite.cpp

```diff
@@ -0,0 +1,52 @@
+#include "../Shell/CmdFullWrite.h"
+#include "../Shell/MockSSD.h"  // �̸��� FakeSSD�� �ٲٴ� �� ��õ!
+#include "gtest/gtest.h"
+
+class FullWriteTest : public ::testing::Test {
+ protected:
+  MockSSD ssd;  // ������ FakeSSD
+  CommandFullWrite* cmd;
+
+  void SetUp() override { cmd = new CommandFullWrite(&ssd); }
+
+  void TearDown() override { delete cmd; }
+};
+
+TEST_F(FullWriteTest, FullWriteSuccess) {
+  std::vector<std::string> program = {"fullwrite", "0xABCDEF12"};
+  EXPECT_TRUE(cmd->Call(program));
+
+  for (int i = 0; i < 100; ++i) {
```

Comment:
lba값이므로 unsigned로 통일하면 좋겠다는 생각이 듭니다!

```suggestion
  for (unsigned int i = 0; i < 100; ++i) {
```